crates/core/tedge_actors/src/errors.rs

```diff
@@ -39,6 +40,9 @@ pub enum RuntimeError {
 
     #[error(transparent)]
     LinkError(#[from] LinkError),
+
+    #[error(transparent)]
+    FileError(#[from] FileError),
```

Comment:
That doesn't look right at this level. There's already an `ActorError` variant in this enum and this `FileError` would have/should have been mapped into an ActorError anyway.